bundles/org.openhab.binding.fsinternetradio/src/test/java/org/openhab/binding/fsinternetradio/test/FSInternetRadioDiscoveryParticipantJavaTest.java

```diff
@@ -181,4 +168,4 @@ private RemoteDevice createUnknownRemoteDevice() throws ValidationException, Mal
         final RemoteService remoteService = null;
         return new RemoteDevice(identity, DEFAULT_TYPE, deviceDetails, remoteService);
     }
-}
+}
```

Comment:
Some of our build tools don't like EOF without a newline. So add a newline here.

Reply:
changed